cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -381,7 +376,7 @@ public static IBuffer GetWindowsRuntimeBuffer(this MemoryStream underlyingStream
             if (length < 0)
                 throw new ArgumentOutOfRangeException(nameof(length));
 
-            if (underlyingStream.Capacity <= positionInStream)
+            if (underlyingStream.Capacity < positionInStream)
```

Comment:
Not sure this is a needed change.

Reply:
I have that one in my test list though (`GetWindowsRuntimeBufferWithZeroCountAtEnd`) and wouldn't expect it to throw, if you first convert via ToArray then it doesn't throw (but makes a copy of the buffer). If the conversion is implicit in the method call it shouldn't throw either.

Reply:
Interesting, I have a test just like that one and it fails but not on this exception, but on the UnauthorizedAccessException check after

Reply:
and yes if I undo this change, that test does fail.